<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveMotor</class>
 <widget class="QDialog" name="SaveMotor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motor Kayıt</string>
  </property>
  <widget class="QLineEdit" name="lineYear">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Renk</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Yıl</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comColor">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Beyaz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Siyah</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kırmızı</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mavi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gri</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineHp">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineModel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Marka</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Motor Gücü</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineBrand">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>821</width>
     <height>151</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Marka</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Model</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Yıl</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Motor Gücü</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Renk</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Soğutma Tipi</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Soğutma Tipi</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comCooler">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Hava</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Su</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yağ</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Ekle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Güncelle</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Yakıt Tüketimi</string>
   </property>
   <widget class="QPushButton" name="btnUpdate_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hesapla</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineKm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kilometre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tüketim</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
